app/routes/pages/components/LandingPage.tsx

```diff
@@ -175,7 +175,7 @@ const LandingPage: PageRenderer = ({ loggedIn }) => {
               extraStyle={{ margin: 0 }}
             />
             <TextWithTitle
-              title="Webkom's besøksadresse"
+              title="Webkom sin besøksadresse"
```

Comment:
```suggestion
              title="Webkoms besøksadresse"
```
This is just preference, either is correct.